code/game/machinery/pipe/construction.dm

```diff
@@ -142,8 +128,6 @@ Buildable meters
 			src.pipe_type = PIPE_PASSIVE_GATE_SUPPLY
 		else if(istype(make_from, /obj/machinery/atmospherics/binary/passive_gate))
 			src.pipe_type = PIPE_PASSIVE_GATE
-		else if(istype(make_from, /obj/machinery/atmospherics/unary/heat_exchanger))
```

Comment:
This one might be an accidental removal?

Reply:
good catch. will fix.